### DISCUSSION TITLE
AF2 predicted UR50 dataset for inverse folding

### DISCUSSION THREAD

**Discussion by EBGU** [facebookresearch/esm#194] [Q&A]
I recently read your inverse folding paper. It is really brilliant! I wonder if it is possible to release your predicted structure of UR50? Thank you very much!

**Comment by tomsercu** [comment]
Thank you!
We are still considering the feasibility of making the large predicted structure dataset available.

**Reply by veghen** [reply]
Would it be possible to only release the 12 million sequences used for structure prediction, selected by running MSA Transformer? Thanks a lost!
